app/src/main/java/io/intrepid/contest/screens/sendinvitations/selectcontacts/SelectContactsActivity.java

```diff
@@ -97,6 +104,26 @@ protected int getLayoutResourceId() {
     }
 
     @Override
+    protected void onViewCreated(Bundle savedInstanceState) {
+        super.onViewCreated(savedInstanceState);
+
+        setActionBarTitle(ACTION_BAR_TITLE);
+        setActionBarDisplayHomeAsUpEnabled(true);
+    }
+
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
+        MenuItem searchItem = menu.add(R.string.common_search);
+        searchItem.setIcon(R.drawable.search_icon);
+        searchItem.setShowAsAction(MenuItem.SHOW_AS_ACTION_IF_ROOM);
+
+        SearchView searchView = new SearchView(this);
+        searchView.setOnQueryTextListener((SearchView.OnQueryTextListener) presenter);
```

Comment:
You can avoid this cast by having the Contract class extend the `OnQueryTextListener` interface instead of the presenter class explicitly implementing it